BLSQ/iaso
# POLIO-1497: SIA Calendar: dotted lines for Test/on hold campaigns

When the tick box “Test campaigns/on hold” is selected on the Campaigns edit view, the campaigns should show on the calendar as dotted lines (or other way if easier, but one should be able to differentiate)

Related JIRA tickets POLIO-1497

## Self proofreading checklist


## Changes

- force categories to all while loading campaigns or calendar
- return is_test on calendar viewset
- display a dashed red line on rounds if campaign is  test/onHold 
- add a small legend to explain the dashed red line
- review style a bit
- moved toggle period buttons to a specific component

## How to test

Open calendar, make sure you have a test campaign, navigate to the round dates and check if red dashed line is present.
We should also test pdf export (only when deployed, i have cross origin problem locally) and embedded calendar

## Print screen / video


<img width="1646" alt="Screenshot 2024-04-08 at 12 14 07" src="https://github.com/BLSQ/iaso/assets/12494624/b1b0bdaf-0bfe-45df-baf9-cbe5f31ec187">
